srcpkgs/ofono/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ofono'
+pkgname=ofono
+version=1.24
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --disable-bluez4"
+conf_files="/etc/ofono/phonesim.conf"
+hostmakedepends="pkg-config"
+makedepends="glib-devel eudev-libudev-devel mobile-broadband-provider-info"
+short_desc="Infrastructure for building mobile telephony (GMS/UTMS) applications"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="GPL-2.0"
+homepage="https://01.org/ofono"
+distfiles="${KERNEL_SITE}/network/${pkgname}/${pkgname}-${version}.tar.xz"
+checksum=9c8e351b7658f4b43f9a4380b731c47d2d7544a89987c48c3f227e73636c87ae
+
+post_install() {
+	vsv ofonod
+	vlicense COPYING
+}
+
+ofono-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vlicense COPYING
```

Comment:
doesn't need to include license for -devel package since it always requires tha main package that has it